ERROR:root:An error occurred in get_advertisement_by_id: 'InstrumentedList' object has no attribute 'wf_state_id'
INFO:sqlalchemy.engine.Engine:SELECT anon_1.advertisement_header_header_id AS anon_1_advertisement_header_header_id, anon_1.advertisement_header_publisher_id AS anon_1_advertisement_header_publisher_id, anon_1.advertisement_header_time_period AS anon_1_advertisement_header_time_period, anon_1.advertisement_header_is_active AS anon_1_advertisement_header_is_active, anon_1.advertisement_header_coordinates AS anon_1_advertisement_header_coordinates, anon_1.advertisement_header_wf_state AS anon_1_advertisement_header_wf_state, anon_1.advertisement_header_date AS anon_1_advertisement_header_date, anon_1.advertisement_header_user_id AS anon_1_advertisement_header_user_id, anon_1.advertisement_header_package_id AS anon_1_advertisement_header_package_id, customer_1.customer_id AS customer_1_customer_id, customer_1.name AS customer_1_name, customer_1.sys_user_id AS customer_1_sys_user_id, advertisement_package_1.package_id AS advertisement_package_1_package_id, advertisement_package_1.description AS advertisement_package_1_description, advertisement_package_1.number_of_add AS advertisement_package_1_number_of_add, advertisement_package_1.num_days AS advertisement_package_1_num_days, advertisement_package_1.is_active AS advertisement_package_1_is_active, advertisement_package_1.price AS advertisement_package_1_price, workflow_history_1.workflow_history_id AS workflow_history_1_workflow_history_id, workflow_history_1.transec_ref_id AS workflow_history_1_transec_ref_id, workflow_history_1.state_id AS workflow_history_1_state_id, workflow_history_1.state AS workflow_history_1_state, workflow_history_1.transection_type AS workflow_history_1_transection_type, advertisement_details_1.advertisement_details_id AS advertisement_details_1_advertisement_details_id, advertisement_details_1.add_id AS advertisement_details_1_add_id, advertisement_details_1.filter_id AS advertisement_details_1_filter_id, advertisement_details_1.value AS advertisement_details_1_value, advertisement_details_1.header_id AS advertisement_details_1_header_id, image_details_1.image_details_id AS image_details_1_image_details_id, image_details_1.image_url AS image_details_1_image_url, image_details_1.transec_ref_id AS image_details_1_transec_ref_id 
FROM (SELECT advertisement_header.header_id AS advertisement_header_header_id, advertisement_header.publisher_id AS advertisement_header_publisher_id, advertisement_header.time_period AS advertisement_header_time_period, advertisement_header.is_active AS advertisement_header_is_active, advertisement_header.coordinates AS advertisement_header_coordinates, advertisement_header.wf_state AS advertisement_header_wf_state, advertisement_header.date AS advertisement_header_date, advertisement_header.user_id AS advertisement_header_user_id, advertisement_header.package_id AS advertisement_header_package_id 
FROM advertisement_header 
WHERE advertisement_header.header_id = %s 
 LIMIT %s) AS anon_1 LEFT OUTER JOIN customer AS customer_1 ON customer_1.customer_id = anon_1.advertisement_header_publisher_id LEFT OUTER JOIN advertisement_package AS advertisement_package_1 ON advertisement_package_1.package_id = anon_1.advertisement_header_package_id LEFT OUTER JOIN workflow_history AS workflow_history_1 ON anon_1.advertisement_header_header_id = workflow_history_1.transec_ref_id LEFT OUTER JOIN advertisement_details AS advertisement_details_1 ON anon_1.advertisement_header_header_id = advertisement_details_1.header_id LEFT OUTER JOIN image_details AS image_details_1 ON anon_1.advertisement_header_header_id = image_details_1.transec_ref_id
INFO:sqlalchemy.engine.Engine:[cached since 141.1s ago] (9, 1)
ERROR:server:Exception on /api/advertisement/quary/get_advertisement_by_id/9 [GET]
Traceback (most recent call last):
  File "d:\QUEE\QueezFlask\queez-flask-api -withblueprints\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\QUEE\QueezFlask\queez-flask-api -withblueprints\venv\Lib\site-packages\flask\app.py", line 1487, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\QUEE\QueezFlask\queez-flask-api -withblueprints\venv\Lib\site-packages\flask\app.py", line 1506, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "d:\QUEE\QueezFlask\queez-flask-api -withblueprints\venv\Lib\site-packages\flask\app.py", line 1801, in make_response
    raise TypeError(
TypeError: The view function for 'advertisement_query_routes.get_advertisement_by_id_endpoint' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 11:05:41] "[35m[1mGET /api/advertisement/quary/get_advertisement_by_id/9 HTTP/1.1[0m" 500 -
